src/common/components/InputField/ErrorMessage/ErrorMessage.tsx

```diff
@@ -0,0 +1,9 @@
+import styles from "./ErrorMessage.module.scss";
+
+interface ErrorMessageProps {
+  errorMessage?: string;
```

Comment:
컴포넌트 이름이 ErrorMessage이기 때문에, 해당 컴포넌트가 보여줄 텍스트는 조금 가볍게 표현해주는게 더 좋을 것 같네요~
```suggestion
  text: string;
```

```suggestion
message: string;
```

Reply:
해당 컴포넌트는 props로 문자열을 받지 못한다면 아무 의미없는 컴포넌트일 것 같은데요~
그러면 props로 받는 문자열이 옵셔널로 표현되는 것은 좋지 않을 것 같습니다~
```suggestion
  errorMessage: string;
```

Reply:
마지막을 고민해볼 포인트는 화면에 보여질 텍스트를 지금과 같이 표현해줄 것인지, children으로 표현해줄것인지 일 것 같네요~

```ts
// 현재 구조
<ErrorMessage errorMessage='에러가 발생했어요!' />

// children 구조
<ErrorMessage>
  에러가 발생했어요!
</ErrorMessage
```